### DISCUSSION TITLE
Adding webp to the convert images tool

### DISCUSSION THREAD

**Discussion by SantosAlarcon** [DevToys-app/DevToys#945] [Ideas]
Hi everyone!

I'm impressed about this tool, but I would like to ask you if it is possible to add the **webp** format to the convert image tool in the **version 2**. And also, if possible, adding a **quality slider** to control the quality of the converted images.

Thank you for this amazing tool.

**Comment by veler** [comment]
Hello,

Supporting WEBP is definitely part of the plan for DevToys 2.0. Not sure about a quality slider though.
